action.yaml

```diff
@@ -22,14 +26,20 @@ runs:
       with:
         elixir-version: ${{ inputs.elixir-version }}
         otp-version: ${{ inputs.erlang-version }}
+    - name: Format Cache Version
+      id: format-cache-version
+      if: ${{ inputs.cache-key-version }}
+      run: echo "::set-output name=cache-key-version::_${{ inputs.cache-key-version }}"
```

Comment:
the key format was done this way in order to keep the current cache keys the same